frontend/src/lib/api/candidate.ts

```diff
@@ -49,7 +49,9 @@ export const me = async (): Promise<any> => {
   return request(
     ['api', 'users', 'me'],
     new URLSearchParams({
-      populate: '*'
+      'populate[candidate][populate][nominations][populate][party]': 'true',
+      'populate[candidate][populate][nominations][populate][constituency]': 'true',
+      'populate[candidate][populate][party]': 'true'
```

Comment:
`'true'` => `true`

Reply:
Other files have used the current syntax and TS complains about it, so I'll leave it as it is for now.